docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -828,7 +828,37 @@ object of the form:
 The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET 
 scans, respectively.
 
-### 5.2 Tar Level Data
+```
+POST /candidates/$CandID/$Visit/dicoms
+```
+
+In addition to the uploaded file, the body of the POST request should contain the
+following keys and values:
+
+```js
+{
+    'CandID': $CandID,
+    'PSCID': string,
+    'Visit': $Visit,
+    'IsPhantom': "true|false"
+}
+```
+
+An optionnal header 'LORIS-Overwrite' with the value 'overwrite' can be used to
+overwrite an existing file. 
```

Comment:
Overwrite the file system only. It always tries to insert in the `mri_upload` table; and because the is not unique contraint other than the auto_increment `ID` column it do not replace.